### PR TITLE
modifierd slurm creation file

### PR DESCRIPTION
**Pull request by SeppiaBrilla**
this should fix the bug of not being able to have the same solver twice with different configuration

### CODE REVIEW

File: scripts/solutions/clean_up.sh [hunk 1]
```diff
@@ -18,4 +18,4 @@ find problems -name '*.eprime-infor'  -delete
 
 # the eprimes are duplicated during solving. remove them here.
 rm -f problems/*/conjure-mode/*/savilerow-mode/*/solver/*/*.eprime
-rm -rf slurm
+rm -rf slurm/sh
```
Comment (ozgurakgun) [inline scripts/solutions/clean_up.sh:21]: did you intend to have a * here?

Comment (SeppiaBrilla) [reply scripts/solutions/clean_up.sh:21]: no. It should create a sub-directory with only the sh files to make the clean-up easier: we will have 3 directories: one for the sh, one for the stdout and one for the stderr files

Comment (ozgurakgun) [reply scripts/solutions/clean_up.sh:21]: oh ok, didn't notice that also changed.

Comment (ozgurakgun) [reply scripts/solutions/clean_up.sh:21]: fyi, I had to make another edit: https://github.com/conjure-cp/EssenceCatalog-runs/commit/83221dc693552b0f2711910a21119f0f60ad9d9a

**Pull request merged**